### ISSUE TITLE
t/01_simple_rest.t fails with non-English locale

### ISSUE THREAD

**Issue by eserte** [JaSei/Test-RestAPI#1]
If a non-English locale is in effect, then the test suite fails. On a debian/linux system:

```
$ env LC_ALL=de_DE.UTF-8 cpan_smoke_modules -pv 5.18.4 JASEI/Test-RestAPI-0.1.1.tar.gz
...
#   Failed test 'after destroy Test object API is dead'
#   at t/01_simple_rest.t line 27.
#     Structures begin differing at:
#          $got->{message} = 'Verbindungsaufbau abgelehnt'
#     $expected->{message} = 'Connection refused'
# Looks like you failed 1 test of 4.
t/01_simple_rest.t ............ 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/4 subtests 
...
```

On other OS the error message may be different. On FreeBSD systems I see:

```
#   Failed test 'after destroy Test object API is dead'
#   at t/01_simple_rest.t line 27.
#     Structures begin differing at:
#          $got->{message} = 'Socket is not connected'
#     $expected->{message} = 'Connection refused'
```

**Comment by eserte** [comment]
Actually, the failure on FreeBSD is something different; the English error message is different here.

**Comment by eserte** [comment]
I see still failures with Test-RestAPI-0.1.4

**Comment by JaSei** [comment]
Thanks for report. Should be fixed with #3 PR.

**Comment by JaSei** [comment]
Released as version 0.1.5
